CHANGES.rst

```diff
@@ -4,6 +4,18 @@ Latest changes
 Development
 -----------
 
+Thomas Moreau
+
+    Limit the number of threads used in worker processes (for ``loky`` and
+    ``multiprocessing``) by C-libraries that relies on threadpools. This
```

Comment:
I would not mention `multiprocessing` as it's not reliable when using the default `fork` context under posix.